src/currentprocess/filesource.rs

```diff
@@ -24,8 +24,8 @@ impl Stdin for io::Stdin {
     fn lock(&self) -> Box<dyn StdinLock + '_> {
         Box::new(io::Stdin::lock(self))
     }
-    fn read_line(&self, buf: &mut String) -> Result<usize> {
-        io::Stdin::read_line(self, buf)
+    fn read_line(&self, buf: &mut str) -> Result<usize> {
+        io::Stdin::read_line(self, &mut buf.to_owned())
```

Comment:
This is confusing, surely that won't work?

Reply:
Yes, this should be a false positive.